Forms/MainForm.cs

```diff
@@ -361,13 +410,30 @@ private void txtCharacterList_TextChanged(object sender, EventArgs e)
             CharacterList charList = new CharacterList();
             charList.AddRange(txtCharacterList.Text.ToArray());
             Configuration.DefaultCharacters = charList;
+            LoadConfigurationDetails();
         }
 
         private void txtWordList_TextChanged(object sender, EventArgs e)
         {
+            int oldWordCount = Configuration.DefaultWords.Count;

```

Comment:
Not sure about the word count to avoid refreshing too much,  I have a commit ready to handle it with the KeyUp event instead, where it only updates after pressing space, backspace, del or enter.  if you want, I can probably push it here?